### ISSUE TITLE
destroy function?

### ISSUE THREAD

**Issue by ghost** [pupunzi/jquery.mb.vimeo_player#21]
is there a way to destroy the player or unload it ?

**Comment by pupunzi** [comment]
No, there's no explicit method to destroy the player once instantiated at the time.
I'll add this method with the next release.

**Comment by ghost** [comment]
that would be great, thank you. i noticed that vimeo recently added destroy functionality to their api
